mso/resource_mso_schema_template_contract.go

```diff
@@ -524,7 +524,11 @@ func resourceMSOTemplateContractRead(d *schema.ResourceData, m interface{}) erro
 	if err != nil {
 		return errorForObjectNotFound(err, d.Id(), schemaCont, d)
 	}
-	setContractFromSchema(d, schemaCont, schemaId, templateName, contractName)
+	err = setContractFromSchema(d, schemaCont, schemaId, templateName, contractName)
+	if err != nil {
+		d.SetId("")
+		log.Printf("[DEBUG] %v", err)
```

Comment:
We need a prefix to this error so we can find it in logs and relate to this specific place.

Reply:
Added prefix